tests/Microsoft.Bot.Builder.Dialogs.Tests/TextPromptTests.cs

```diff
@@ -113,8 +113,8 @@ public async Task TextPromptWithNaughtyStrings()
                 {
                     // If the input message is empty after a .Trim() operation, character the comparison will fail because the reply message will be a
                     // Message Activity with null as Text, this is expected behavior
-                    var messageIsBlank = e.Message.Equals(" :\nExpected: \nReceived:", StringComparison.CurrentCultureIgnoreCase) && naughtyString.Equals(" ", StringComparison.CurrentCultureIgnoreCase);
-                    var messageIsEmpty = e.Message.Equals(":\nExpected:\nReceived:", StringComparison.CurrentCultureIgnoreCase) && naughtyString.IsNullOrEmpty();
+                    var messageIsBlank = e.Message.Equals(" :\nExpected: \nReceived:", StringComparison.Ordinal) && naughtyString.Equals(" ", StringComparison.Ordinal);
+                    var messageIsEmpty = e.Message.Equals(":\nExpected:\nReceived:", StringComparison.Ordinal) && naughtyString.IsNullOrEmpty();
```

Comment:
Same as mentioned previously, CurrentCultureIgnoreCase to OrdinalIgnoreCase

Reply:
Fixed. Thanks!